open_rarity/scoring/scorer.py

```diff
@@ -42,24 +62,63 @@ def score_tokens(
             collection (Collection): The collection to score from
             tokens (list[Token]): a batch of tokens belonging to collection
                 to be scored
-            normalized (bool, optional): _description_. Defaults to True.
```

Comment:
should we remove normalized flag from the scorer interface? Perhaps passing an optional wrapper for scoring flag is a better way to pass configs for individual scorers

Reply:
not sure what you mean by this, but i feel like this interface is fine